File: artifacts/lodepng/decompose/src/lode_p_n_g_decompress_settings.c
struct LodePNGDecompressSettings
{
  unsigned ignore_adler32;
  unsigned ignore_nlen;
  size_t max_output_size;
  unsigned (*custom_zlib)(unsigned char **, size_t *, const unsigned char *, size_t, const LodePNGDecompressSettings *);
  unsigned (*custom_inflate)(unsigned char **, size_t *, const unsigned char *, size_t, const LodePNGDecompressSettings *);
  const void *custom_context;
}
/*** DEPENDENCIES:
***/

File: artifacts/lodepng/translate_ablation_1/src/lode_p_n_g_decompress_settings.rs
use crate::*;
use std::os::raw::c_uint;
use std::os::raw::c_void;


#[derive(Clone)]
pub struct LodePngDecompressSettings {
    pub ignore_adler32: c_uint,
    pub ignore_nlen: c_uint,
    pub max_output_size: usize,
    pub custom_zlib: Option<fn(&mut *mut u8, &mut usize, &u8, usize, &LodePngDecompressSettings) -> c_uint>,
    pub custom_inflate: Option<fn(&mut *mut u8, &mut usize, &u8, usize, &LodePngDecompressSettings) -> c_uint>,
    pub custom_context: Option<*const c_void>,
}

